lib/iris/common/metadata.py

```diff
@@ -1344,6 +1344,9 @@ def __init__(self, cls, **kwargs):
         #: The metadata class to be manufactured by this factory.
         self.cls = cls
 
+        # Proxy for self.cls._fields
+        self._fields = cls._fields
+
```

Comment:
Probably worth a comment in the code here to explain whey you're creating a proxy and then on the next line not using it!